cmd/vic-machine/create/create.go

```diff
@@ -1257,12 +1250,30 @@ func (c *Create) Run(cliContext *cli.Context) (err error) {
 	log.Info("")
 
 	executor := management.NewDispatcher(ctx, validator.Session, vchConfig, c.Force)
+
 	if err = executor.CreateVCH(vchConfig, vConfig); err != nil {
 		executor.CollectDiagnosticLogs()
 		log.Error(err)
 		return err
 	}
 
+	// after create appliance, use timeout context to wait ip address
+	timeoutctx, cancel := context.WithTimeout(context.Background(), c.Timeout)
+	defer cancel()
+	defer func() {
+		if timeoutctx.Err() != nil && timeoutctx.Err() == context.DeadlineExceeded {
+			//context deadline exceeded, replace returned error message
+			err = errors.Errorf("Create timed out: if slow connection, increase timeout with --timeout")
```

Comment:
Displaying the number of seconds for timeout would be very useful, since user may not have any idea what the default timeout is.
Also, If I was a user, I would be happier to see message like this. 

Creating VCH exceeded time limit of %s seconds. Please, increase timeout using --time in case of slow network bandwidth or busy vSphere target.